flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -204,6 +204,15 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
                 #endif
             }
 
+            if let sdk = arguments["sdk"] as? [String: Any?],
+               let name = sdk["name"] as? String,
+               let version = sdk["version"] as? String {
+//                let sdkVersion = SdkVersion(name, version)
+//                options.setSentryClientName(name)
+//                options.setSdkVersion(sdkVersion)
+               PrivateSentrySDKOnly.setSdkName(name, andVersionString: version)
```

Comment:
Check if the fields are not null as well, similar to Kotlin

Reply:
We do this with consecutive `if let`, or do you mean something else?

<img width="663" alt="Bildschirm­foto 2023-01-16 um 11 11 24" src="https://user-images.githubusercontent.com/3984453/212653179-594420ca-3d64-47bd-b8ea-fd86a2cf14d5.png">